servers/rendering/renderer_scene_cull.cpp

```diff
@@ -1160,7 +1160,8 @@ void RendererSceneCull::_update_instance(Instance *p_instance) {
 
 	p_instance->transformed_aabb = new_aabb;
 
-	if (p_instance->scenario == nullptr || !p_instance->visible || Math::is_zero_approx(p_instance->transform.basis.determinant())) {
+	// note: is equal approx means 0.01 scale won't work so don't do that.
```

Comment:
That sounds weird, `is_zero_approx` uses `CMP_EPSILON` which is `0.00001`, that's 1000 times smaller than `0.01`.

Reply:
Smallest value which is valid to render are these:

so these aren't precision errors
```cpp
Smallest rendered value is 0.000004
Smallest rendered value is 0.000004
Smallest rendered value is 0.000004
Smallest rendered value is 0.000004
Smallest rendered value is 0.000004
Smallest rendered value is 0.000004
Smallest rendered value is 0.000003
```

determinant is returning these, but they're valid.

float can do much much smaller numbers albeit with precision errors but the value must be dealt with otherwise the rendered scene breaks.

Reply:
corrected